src/ribasim_nl/ribasim_nl/analyse_results.py

```diff
@@ -404,7 +413,7 @@ def CompareOutputMeasurements(
             stats=stats,
             koppelinfo=full_title,
             fig_name=fig_name,
-            bron_meting=meetlocaties_link.iloc[0]["Waterschap"],
+            bron_meting=None if apply_for_water_authority is not None else meetlocaties_link.iloc[0]["Waterschap"],
```

Comment:
Can you move `bron_meting = None if apply_for_water_authority is not None else meetlocaties_link.iloc[0]["Waterschap"]` up and use `bron_meting = bron_meting` here and below?